src/SamplesApp/UITests.Shared/Windows_UI_Xaml/Performance/Performance_Dopes.xaml.cs

```diff
@@ -0,0 +1,255 @@
+							avgSum += r;
+							avgN++;
+						}
+
+						prevMs = sw.ElapsedMilliseconds;
+					}
+				}
+
+				_ = Dispatcher.RunAsync(Windows.UI.Core.CoreDispatcherPriority.Low, () => loop());
+			};
+
+			sw.Start();
+
+			_ = Dispatcher.RunAsync(Windows.UI.Core.CoreDispatcherPriority.Normal, () => loop());
+		}
+
+		private void StartTestReuseST()
+		{
+			var rand = new Random(0);
+
+			breakTest = false;
+
+			var width = absolute.ActualWidth;
+			var height = absolute.ActualHeight;
+
+			const int step = 20;
+			var labels = new TextBlock[step * 2];
+
+			var processed = 0;
+
+			long prevTicks = 0;
+			long prevMs = 0;
+			int prevProcessed = 0;
+			double avgSum = 0;
+			int avgN = 0;
+			var sw = new Stopwatch();
+
+			Action loop = null;
+
+			System.Collections.Concurrent.ConcurrentBag<TextBlock> _cache = new System.Collections.Concurrent.ConcurrentBag<TextBlock>();
+
+			loop = () =>
+			{
+				var now = sw.ElapsedMilliseconds;
+
+				if (breakTest)
+				{
+					var avg = avgSum / avgN;
+					dopes.Text = string.Format("{0:0.00} Dopes/s (AVG)", avg).PadLeft(21);
+					return;
+				}
+
+				//60hz, 16ms to build the frame
+				while (sw.ElapsedMilliseconds - now < 16)
+				{
+					if (!_cache.TryTake(out var label))
+					{
+						label = new TextBlock();
+					}
+
+					label.Text = "Dope";
+					label.Foreground = new SolidColorBrush(Color.FromArgb(0xFF, (byte)(rand.NextDouble() * 255), (byte)(rand.NextDouble() * 255), (byte)(rand.NextDouble() * 255)));
+
+					label.RenderTransform = new RotateTransform() { Angle = rand.NextDouble() * 360 };
+
+					Canvas.SetLeft(label, rand.NextDouble() * width);
+					Canvas.SetTop(label, rand.NextDouble() * height);
+
+					if (processed > max)
+					{
+						_cache.Add(absolute.Children[0] as TextBlock);
+						absolute.Children.RemoveAt(0);
+					}
+
+					absolute.Children.Add(label);
+
+					processed++;
+
+					if (sw.ElapsedMilliseconds - prevMs > 500)
+					{
+
+						var r = (double)(processed - prevProcessed) / ((double)(sw.ElapsedTicks - prevTicks) / Stopwatch.Frequency);
+						prevTicks = sw.ElapsedTicks;
+						prevProcessed = processed;
+
+						if (processed > max)
+						{
+							dopes.Text = string.Format("{0:0.00} Dopes/s", r).PadLeft(15);
+							avgSum += r;
+							avgN++;
+						}
+
+						prevMs = sw.ElapsedMilliseconds;
+					}
+				}
+
+				_ = Dispatcher.RunAsync(Windows.UI.Core.CoreDispatcherPriority.Low, () => loop());
+			};
+
+			sw.Start();
+
+			_ = Dispatcher.RunAsync(Windows.UI.Core.CoreDispatcherPriority.Low, () => loop());
+		}
+
+		private void SetControlsAtStart()
+		{
+			startChangeST.Visibility = startST.Visibility = startGridST.Visibility = Visibility.Collapsed;
+			stop.Visibility = dopes.Visibility = Visibility.Visible;
+			absolute.Children.Clear();
+			grid.Children.Clear();
+			dopes.Text = "Warming up..";
+		}
+
+		private void startMT_Clicked(System.Object sender, object e)
+		{
+			SetControlsAtStart();
+			//StartTestMT2();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5700024)